lib/features.js

```diff
@@ -83,5 +83,8 @@ module.exports = {
     generators: false,
 
     // React JSX parsing
-    jsx: false
+    jsx: false,
+
+    // enable parsing of import/export declarations
+    module: false
```

Comment:
I think it's fine to add an `isModule` flag so you don't need to keep comparing `scriptType`. It's just having an `ecmaFeature` of `module` doesn't make much sense since it can't be independently controlled outside of `scriptType`'.